index.html

```diff
@@ -67,207 +67,262 @@ <h2>LET'S CONNECT</h2>
-              <li>
-                <a href="#">
-                  <img src="image/Linkedin_icon.png" class="icon" alt="linkedin"/>
-                </a>
-              </li>
-              <li>
-                <a href="https://google.com">
-                  <img src="image/angellist.png" class="icon" alt="google"/>
-                </a>
+        <div class="headline-main-container1">
+          <div class="main-headline1">
+            <h3 class="welcome">
+              About <br />
+                Myself
+            </h3>
+            <p class="introduction">
+              Hello I’m a software developer! I can help you build a product ,
+              feature or website Look through some of my work and experience! If
+              you like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+            <div class="link-up">
+              <h2>LET'S CONNECT</h2>
+              <div class="social-list"></div>
+                <ul>
+                  <li>
+                    <a href="#">
+                      <img src="image/github.png" class="icon" alt="git"/>
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#">
+                      <img src="image/Linkedin_icon.png" class="icon" alt="linkedin"/>
+                    </a>
+                  </li>
+                  <li>
+                    <a href="https://google.com">
+                      <img src="image/angellist.png" class="icon" alt="google"/>
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#">
+                      <img src="image/twitter.png" class="icon" alt="twitter"/>
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#">
+                      <img src="image/medium.png" class="icon" alt="mediamarket"/>
+                    </a>
+                  </li>
+                </ul>
+                <div class="btn-T button">
+                  <button type="button">Get my resume</button>
+                </div>
+              </div>
+          </div>
+          <div class="main-knowledges">
+            <ul class="knowledges">
+              <li class="languages knw">
+                <div class="lang">
+                  <h3>Langguages</h3>
+                  <i class="fas fa-chevron-down"></i>
+                </div>
+                <ul>
+                  <li>
+                    <img src="image/css.svg" class="img-list" alt="javascript"/>
+                    <pre>javaScript</pre>
+                  </li>
+                  <li>
+                    <img src="image/js.svg" class="img-list" alt="html"/>
+                    <pre>html</pre>
+                  </li>
+                  <li>
+                    <img src="image/html.svg" class="img-list" alt="css"/>
+                    <pre>css</pre>
+                  </li>
+                </ul>
```

Comment:
Style the text below the image in the language as shown in the difference below
### Original design
![right](https://user-images.githubusercontent.com/58376020/126869229-0c97e85b-2926-466e-a561-8ac278538f38.png)

#### Your Project
![wrong](https://user-images.githubusercontent.com/58376020/126869239-f7c1b76c-87a2-4a7e-b22b-b4c4fa697779.png)